theme/less/main.less

```diff
@@ -1,23 +1,26 @@
 @import "_variables.less";
 @import "_navigation.less";
+@import "_header.less";
+@import "_grid.less";
 
 body
 {
-  margin: 0px;
-  font-family: sans-serif;
+  font-family: Roboto, sans-serif;
+  margin: 0;
 }
 
-header
-{
-  background-color: @primary;
-  padding: 30px;
+@font-face {
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 500;
```

Comment:
I feel like that the weight visually looks thinner in the Figma, let's make the font looks similar to those in figma.

Reply:
Meeting-note: We will try using `400` to see if it fits more.